### PR TITLE
Update CI definitions + add @Dependabot

### PR DESCRIPTION
**Pull request by Sija**

### CODE REVIEW

**Comment by Sija** [comment]
ftr, the CI failures are unrelated.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
```
Comment (veelenga) [inline .github/dependabot.yml:7]: i would do weekly. Daily is too often IMO

Comment (Sija) [reply .github/dependabot.yml:7]: Considering the (tiny) amount of actions to bump, I don't think it matters.

**Review by veelenga — APPROVED** [review]
LGTM

**Pull request merged**